packages/react-tooltip/src/components/Tooltip/private/constants.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * The height of the tooltip's arrow in pixels.
+ */
+export const arrowHeight = 6;
+
+/**
+ * The default value of the tooltip's border radius (borderRadiusMedium).
+ *
+ * Unfortunately, Popper requires it to be specified as a variable instead of using CSS.
+ * While we could use getComputedStyle, that adds a performance penalty for something that
+ * will likely never change.
+ */
+export const tooltipBorderRadius = 4;
```

Comment:
Now that tokens are exported does it make sense for this to be

```suggestion
export const tooltipBorderRadius = tokens.borderRadiusMedium;
```

so that you can use this same value here, in `useTooltip` and in lie 21 of `useTooltipStyles`?

Reply:
`tokens.borderRadiusMedium` is a css variable, right? As far as I know, Popper requires a number.

Reply:
Ahh, you're right, disregard my comment then.